ruby/red-parquet/test/test-arrow-file-reader.rb

```diff
@@ -0,0 +1,56 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+class TestArrowFileReader < Test::Unit::TestCase
+  def setup
+    visible_field = Arrow::Field.new("visible", :boolean)
+    @schema = Arrow::Schema.new([visible_field])
+    visible_arrays = [Arrow::BooleanArray.new([true, false])]
+    visible_array = Arrow::ChunkedArray.new(visible_arrays)
+    table = Arrow::Table.new(@schema, [visible_array])
+    @file = Tempfile.open(["red-parquet", ".parquet"])
+    chunk_size = 1
+    writer = Parquet::ArrowFileWriter.new(table.schema, @file.path)
+    writer.write_table(table, chunk_size)
+    writer.close
```

Comment:
Could you use more Ruby-ish `ArrowFileWriter.open {...}` API?

```suggestion
    Parquet::ArrowFileWriter.open(table.schema, @file.path) do |writer|
      chunk_size = 1
      writer.write_table(table, chunk_size)
    end
```

Reply:
fix: [4c228dd](https://github.com/apache/arrow/pull/36022/commits/4c228dd85612dcb8efc785427b1c798a205b5128)